gulpfile.js

```diff
@@ -217,7 +217,7 @@ function librarySizeTask() {
 function moduleSizesTask() {
   return gulp.src(srcDir + '**/*.js')
     .pipe(uglify({
-      preserveComments: 'some'
+      comments: 'some'
```

Comment:
That doesn't work (`GulpUglifyError: unable to minify JavaScript`) but you were right in the first place, you can remove it. I didn't notice it was `moduleSizesTask`, I tought it was the `buildTask`, for which we insert the header *after* minifying.